### PR TITLE
Use latest Broccoli

### PR DESCRIPTION
**Pull request by oligriffiths**
Convert to the latest broccoli syntax.
Will update to a tag release once we tag

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +25,7 @@
     "walk-sync": "^0.3.3"
   },
   "devDependencies": {
-    "broccoli": "^2.0.1",
+    "broccoli": "broccolijs/broccoli#master",
```
Comment (stefanpenner) [inline package.json]: should we wait for release?

Comment (oligriffiths) [reply package.json]: Yup. I want to cut w release soon with the BROCCOLI_ENV stuff in it.

Comment (oligriffiths) [reply package.json]: V2.1.0

File: lib/plugin/printer.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const Funnel = require('broccoli-funnel');
+const glob = require('glob').sync;
+const fs = require('fs');
+
+class Printer extends Funnel
```
Comment (stefanpenner) [inline lib/plugin/printer.js]: Funnel?

File: lib/plugin/printer.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+const Funnel = require('broccoli-funnel');
+const glob = require('glob').sync;
```
Comment (stefanpenner) [inline lib/plugin/printer.js]: `walkSync`

**Pull request merged**